packages/jellyfish-api-core/src/category/loan.ts

```diff
@@ -370,5 +392,13 @@ export interface DepositVault {
 
 export interface TakeLoanMetadata {
   vaultId: string
-  amounts: string // amount@symbol
+  amounts: string | string[] // amount@symbol
+  to?: string
```

Comment:
Add a `takeloan` test to cover this

Reply:
nm. you have it in `packages/jellyfish-api-core/__tests__/category/loan/auctionBid.test.ts`

Reply:
its on https://github.com/DeFiCh/jellyfish/pull/699